tests/components/schedule/test_init.py

```diff
@@ -177,6 +176,55 @@ async def test_invalid_schedules(
     assert error in caplog.text
 
 
+@pytest.mark.parametrize(
+    "schedule",
+    ({CONF_FROM: "07:00:00", CONF_TO: "11:00:00"},),
```

Comment:
Why is this parametrized? it only contains a single item

Reply:
Fixed by https://github.com/home-assistant/core/pull/77696